samples/config/minimal_config.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
Minimal configuration sample, not to overwhelm newcomers with huge configuration files.

Reply:
Minimal example for "start" and "join".